public/components/wz-menu/wz-menu-agent.js

```diff
@@ -44,32 +44,54 @@ class WzMenuAgent extends Component {
         id: 'regulatoryCompliance',
         text: 'Regulatory Compliance'
       },
-      general: { id: 'general', text: 'Security Events' },
-      fim: { id: 'fim', text: 'Integrity Monitoring' },
-      aws: { id: 'aws', text: 'Amazon AWS' },
-      gcp: { id: 'gcp', text: 'Google Cloud Platform' },
-      pm: { id: 'pm', text: 'Policy Monitoring' },
-      sca: { id: 'sca', text: 'Security configuration assessment' },
-      audit: { id: 'audit', text: 'System Auditing' },
-      oscap: { id: 'oscap', text: 'OpenSCAP' },
-      ciscat: { id: 'ciscat', text: 'CIS-CAT' },
-      vuls: { id: 'vuls', text: 'Vulnerabilities' },
-      virustotal: { id: 'virustotal', text: 'VirusTotal' },
-      osquery: { id: 'osquery', text: 'Osquery' },
-      docker: { id: 'docker', text: 'Docker Listener' },
-      mitre: { id: 'mitre', text: 'MITRE ATT&CK' },
-      pci: { id: 'pci', text: 'PCI DSS' },
-      gdpr: { id: 'gdpr', text: 'GDPR' },
-      hipaa: { id: 'hipaa', text: 'HIPAA' },
-      nist: { id: 'nist', text: 'NIST 800-53' },
-      tsc: { id: 'tsc', text: 'TSC' }
+      general: { id: WAZUH_MODULES_ID.SECURITY_EVENTS, text: 'Security Events' },
+      fim: { id: WAZUH_MODULES_ID.INTEGRITY_MONITORING, text: 'Integrity Monitoring' },
+      aws: { id: WAZUH_MODULES_ID.AMAZON_WEB_SERVICES, text: 'Amazon AWS' },
+      gcp: { id: WAZUH_MODULES_ID.GOOGLE_CLOUD_PLATFORM, text: 'Google Cloud Platform' },
+      pm: { id: WAZUH_MODULES_ID.POLICY_MONITORING, text: 'Policy Monitoring' },
+      sca: { id: WAZUH_MODULES_ID.SECURITY_CONFIGURATION_ASSESSMENT, text: 'Security configuration assessment' },
+      audit: { id: WAZUH_MODULES_ID.AUDITING, text: 'System Auditing' },
+      oscap: { id: WAZUH_MODULES_ID.OPEN_SCAP, text: 'OpenSCAP' },
+      ciscat: { id: WAZUH_MODULES_ID.CIS_CAT, text: 'CIS-CAT' },
+      vuls: { id: WAZUH_MODULES_ID.VULNERABILITIES, text: 'Vulnerabilities' },
+      virustotal: { id: WAZUH_MODULES_ID.VIRUSTOTAL, text: 'VirusTotal' },
+      osquery: { id: WAZUH_MODULES_ID.OSQUERY, text: 'Osquery' },
+      docker: { id: WAZUH_MODULES_ID.DOCKER, text: 'Docker Listener' },
+      mitre: { id: WAZUH_MODULES_ID.MITRE_ATTACK, text: 'MITRE ATT&CK' },
+      pci: { id: WAZUH_MODULES_ID.PCI_DSS, text: 'PCI DSS' },
+      gdpr: { id: WAZUH_MODULES_ID.GDPR, text: 'GDPR' },
+      hipaa: { id: WAZUH_MODULES_ID.HIPAA, text: 'HIPAA' },
+      nist: { id: WAZUH_MODULES_ID.NIST_800_53, text: 'NIST 800-53' },
+      tsc: { id: WAZUH_MODULES_ID.TSC, text: 'TSC' }
```

Comment:
Great job with all of these constants! 👏🏼 
Maybe we can create constants for the `text` values.

Reply:
Yes, you are right, we should move them to constants too. These `text` values are using in more files, we should refactor them in the next patches.